ipa-core/src/bin/helper.rs

```diff
@@ -185,11 +185,14 @@ async fn server(args: ServerArgs, logging_handle: LoggingHandle) -> Result<(), B
     let shard_network_config = NetworkConfig::new_shards(vec![], shard_clients_config);
     let (shard_transport, _shard_server) = ShardHttpTransport::new(
         IpaRuntime::from_tokio_runtime(&http_runtime),
-        ShardIndex::FIRST,
+        Sharded {
```

Comment:
something like `Sharded::with_shards(1)` could be more concise to read

Reply:
This initialization to be the single shard is temporary. I created a `new` function to help with the conciseness.